Cod3rsGrowth.Servico/Servicos/ServicoProduto.cs

```diff
@@ -25,8 +26,19 @@ public void Adicionar(Produto produto)
 
         public void Atualizar(Produto produto)
         {
-            _produtoValidacao.ValidateAndThrow(produto);
-            _repositorioProduto.Atualizar(produto);
+            try
+            {
+                _produtoValidacao.ValidateAndThrow(produto);
+                _repositorioProduto.Atualizar(produto);
+            }
+            catch (ValidationException ve)
```

Comment:
verifica se a classe de exceção que o fluent retorna é uma exceção verificada, se for, vc não precisa de dois catch